README.md

```diff
@@ -36,7 +36,7 @@ Note on NVME storage: Samsung PM981 drives will not work out of the box if at al
 
 ### Working
 
-- [x] Keyboard. Volume and brightness control keys
+- [x] Keyboard (supported: volume, keyboard backlight and display brightness keys)
```

Comment:
this reads like every other key isn't supported
change "supported" to "including" maybe?